_data/learnindex.json

```diff
@@ -1950,414 +1950,483 @@
                 "author": "Unfolding Defold",
                 "description": "This video explains the Messaging and Addressing system in Defold.",
                 "name": "How does Messaging and Addressing Systems work in Defold?",
-                "path": "https://www.youtube.com/embed/DT4D0d47udA"
+                "embed": "https://www.youtube.com/embed/DT4D0d47udA",
```

Comment:
The learnindex.json is generated from en.json in defold/doc:

https://github.com/defold/doc/blob/master/docs/en/en.json#L2-L14

You need to change the source and then run `update.py --download docs`

Reply:
Makes sense. Updated the docs in https://github.com/defold/doc/pull/480. Will run the update.py once it's merged.